events/blocks/events-form/events-form.js

```diff
@@ -565,6 +576,8 @@ export default async function decorate(block, formData = null) {
     form: block.querySelector(':scope > div:nth-of-type(2) a[href$=".json"]'),
     terms: block.querySelector(':scope > div:nth-of-type(3)'),
     successMsg: block.querySelector(':scope > div:last-of-type > div'),
+    title: block.querySelector(':scope #event-title'),
```

Comment:
I probably wouldn't do this only because the ID is generated and can be changed by authoring. Most other places when we want event title we do `getMetadata('event-title')`. Maybe you can do that too in the `eventFormSendAnalytics` and save one line here?